tutorials/plot_spatial.py

```diff
@@ -11,40 +11,237 @@
 # Apply spatial measures to spatial data
 # --------------------------------------
 #
-# Words, words, words.
+# Sections
+# ~~~~~~~~
+#
+# This tutorial contains the following sections:
+#
+# 1. Compute and plot distances and speed using position
+# 2. Divide position in spatial bin edges
+# 3. Compute spatial bin assignment using spatial bin edges
+# 4. Compute time in each timestamp sample
+# 5. Compute and plot occupancy
+# 6. Compute 2D and 1D spatial information
 #
 
 ###################################################################################################
 
 # sphinx_gallery_thumbnail_number = 1
 
-import spiketools
+# import auxiliary libraries
+import numpy as np
+import seaborn as sns
+import matplotlib.pyplot as plt
+
+# import all functions from spiketools.spatial
+from spiketools.spatial.position import *
+from spiketools.spatial.occupancy import *
+from spiketools.spatial.utils import *
+from spiketools.spatial.information import *
+
+# import sim_spiketrain_binom to simulate spiketrain
+from spiketools.sim.dist import sim_spiketrain_binom
+
+###################################################################################################
+
+# Set some tracking data
+x_pos = np.linspace(0, 15, 16)
+y_pos = np.array([0, 0, 0.1, 1, 1.5, 1, 1, 2.1, 2, 1, 0, 1, 2, 3, 4, 3.2])
+position = np.array([x_pos, y_pos])
+
+# Set the time width of each position sample
+bin_widths = np.array([1, 1, 1, 2, 1.5, 1, 1, 1, 2, 1, 1, 1, 1, 1, 0.5])
+
+# Set number of spatial bins, 3 x-bins and 5 y-bins
+bins = [3, 5]
+
+# Set timestamp
+timestamps = np.array([0, 1, 2, 3, 5, 6.5, 7.5, 8.5, 9.5, 11.5, 12.5, 13.5, 14.5, 15.5, 16.5, 17])
+
+###################################################################################################
+# 1. Compute and plot distances and speed using position
+# ~~~~~~~~
+#
+# Use x- and y-position to compute distance between two or more points, cumulative distance 
+# traveled and speed
+#
+
+###################################################################################################
+
+# Look at the range of our x and y positions
+ranges = get_pos_ranges(position)
+print(f'The x-position ranges from {ranges[0][0]} to {ranges[0][1]}')
+print(f'The y-position ranges from {ranges[1][0]} to {ranges[1][1]}')
+
+# Plot positions (coordinates marked x are the actual points)
+plt.plot(x_pos, y_pos, '-x', c = 'tab:gray', markersize = 10)
```

Comment:
I think this can be replaced by using the `plot_positions` function from `spiketools.plts.space`. 

There might be some things we want / need to extend in this plot, for example adding the x's, and adding axis labels. We can extend this in the plot function, and then use this plot throughout the tutorial.